JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -44,117 +46,116 @@ import com.example.jetnews.model.Post
         }
     }
 }
 
 @Composable
 private fun HomeScreenTopSection(post: Post) {
-ProvideEmphasis(emphasis = EmphasisLevels().high) {
+    ProvideEmphasis(EmphasisAmbient.current.high) {
         Text(
-            modifier = LayoutPadding(start = 16.dp, top = 16.dp, end = 16.dp, bottom = 0.dp),
+            modifier = Modifier.padding(start = 16.dp, top = 16.dp, end = 16.dp, bottom = 0.dp),
             text = "Top stories for you",
-            style = MaterialTheme.typography().subtitle1
+            style = MaterialTheme.typography.subtitle1
         )
     }
-    Ripple(bounded = true) {
-        Clickable(onClick = {
-            navigateTo(Screen.Article(post.id))
-        }) {
-            PostCardTop(post = post)
-        }
+    Clickable(modifier = Modifier.ripple(), onClick = { navigateTo(Screen.Article(post.id)) }) {
+        PostCardTop(post = post)
     }
     HomeScreenDivider()
 }
 
 @Composable
 private fun HomeScreenSimpleSection(posts: List<Post>) {
-    posts.forEach { post ->
-        PostCardSimple(post)
-        HomeScreenDivider()
+    Column {
+        posts.forEach { post ->
+            PostCardSimple(post)
+            HomeScreenDivider()
+        }
     }
 }
 
 @Composable
 private fun HomeScreenPopularSection(posts: List<Post>) {
-    ProvideEmphasis(emphasis = EmphasisLevels().high) {
-        Text(
-            modifier = LayoutPadding(16.dp),
-            text = "Popular on Jetnews",
-            style = MaterialTheme.typography().subtitle1
-        )
-    }
-    HorizontalScroller {
-        Row(modifier = LayoutPadding(top = 0.dp, start = 0.dp, end = 16.dp, bottom = 16.dp)) {
-            posts.forEach { post ->
-                PostCardPopular(modifier = LayoutPadding(start = 16.dp), post = post)
+    Column {
+        ProvideEmphasis(EmphasisAmbient.current.high) {
+            Text(
+                modifier = Modifier.padding(16.dp),
+                text = "Popular on Jetnews",
+                style = MaterialTheme.typography.subtitle1
+            )
+        }
+        HorizontalScroller {
+            Row(modifier = Modifier.padding(start = 0.dp, top = 0.dp, end = 16.dp, bottom = 16.dp)) {
+                posts.forEach { post ->
+                    PostCardPopular(post, Modifier.padding(start = 16.dp))
+                }
             }
         }
+        HomeScreenDivider()
     }
-    HomeScreenDivider()
 }
 
 @Composable
 private fun HomeScreenHistorySection(posts: List<Post>) {
-    posts.forEach { post ->
-        PostCardHistory(post)
-        HomeScreenDivider()
+    Column {
+        posts.forEach { post ->
+            PostCardHistory(post)
+            HomeScreenDivider()
+        }
     }
 }
 
 @Composable
 private fun HomeScreenDivider() {
-    Opacity(0.08f) {
-        Divider(modifier = LayoutPadding(start = 14.dp, top = 0.dp, end = 14.dp, bottom = 0.dp))
-    }
+    Divider(
+        modifier = Modifier.padding(start = 14.dp, top = 0.dp, end = 14.dp, bottom = 0.dp),
```

Comment:
Omit the top/bottom as 0dp is the default?

Reply:
Thanks, applied universally to the rest of the source too since this should always be the case.